packages/commercetools/theme/assets/ui-state.js

```diff
@@ -0,0 +1,17 @@
+import Vue from 'vue'
+import VueCompositionApi, { reactive, ref, computed } from '@vue/composition-api'
+
+Vue.use(VueCompositionApi);
+
+const UiState = reactive({
+  isCartSidebarOpen: false
+})
+
+const isCartSidebarOpen = computed({
+  get: () => UiState.isCartSidebarOpen,
+  set: val => { UiState.isCartSidebarOpen = val }
```

Comment:
I think it's safer to have a toggle method. Then user can override this variable, just change its value